minidump-stackwalk/http_symbol_supplier.cc

```diff
@@ -236,6 +236,20 @@ namespace {
   string JoinURL(CURL* curl, const string& url, const string& sub) {
     return url + "/" + URLEncode(curl, sub);
   }
+
+  string GetCodeIDAndFileQueryString(CURL* curl, const CodeModule* module) {
+      string code_file = PathnameStripper::File(module->code_file());
+      if (code_file.empty()) {
+        return "";
+      }
+
+      string code_id = PathnameStripper::File(module->code_identifier());
+      if (code_id.empty()) {
+        return "";
+      }
+
+      return "?code_file=" + URLEncode(curl, code_file) + "&code_id=" + URLEncode(curl, code_id);
```

Comment:
Gotcha! Yeah, because we use the URL "as is" when assuming the file URL. 

Tecken does this differently. It deconstructs the URL to extract out the domain, bucket name and prefix. And other config variables are extracted from actually parsing the query string. 
https://github.com/mozilla-services/tecken/blob/master/tecken/base/symboldownloader.py#L82-L85